src/components/QueuedTransactionList/index.tsx

```diff
@@ -0,0 +1,167 @@
+): Promise<TransactionSummary[]> => {
+  const { results } = await getTransactionQueue(GATEWAY_URL, chainId, checksumAddress(safeAddress))
+
+  // If there are no queued transactions
+  if (!results.length) return []
+
+  const awaitingConfirmationTxs = results
+    .filter(isTransactionType)
+    .filter(
+      (result) =>
+        result.transaction.txStatus === 'AWAITING_CONFIRMATIONS' &&
+        result.transaction.executionInfo?.type === 'MULTISIG' &&
+        result.transaction.executionInfo.missingSigners?.some((addr) => addr.value === account),
+    )
+    .map(({ transaction }) => transaction)
+
+  return awaitingConfirmationTxs
+}
+
+const TxsToConfirmList = (): ReactElement => {
+  const ownedSafes = useOwnerSafes()
+  const userAccount = useSelector(userAccountSelector)
+
+  const [loading, setLoading] = useState<boolean>(true)
+  const [txsAwaitingConfirmation, setTxsAwaitingConfirmation] = useState<Record<ChainId, TransactionSummary[]>>({})
+
+  // Fetch txs awaiting confirmations after retrieving the owned safes from the LocalStorage
+  useEffect(() => {
+    if (!Object.keys(ownedSafes || {}).length) {
+      return
+    }
+    const fetchAwaitingConfirmationTxs = async () => {
+      const txs: Record<string, TransactionSummary[]> = {}
+
+      for (const [chainId, safesPerChain] of Object.entries(ownedSafes)) {
+        const arrayPromises: Array<Promise<TransactionSummary[]>> = []
+        for (const safe of safesPerChain) {
+          arrayPromises.push(getTxsAwaitingYourSignatureByChainId(chainId, safe, userAccount))
+        }
+        const txsByChain: TransactionSummary[] = await Promise.all(arrayPromises).then((values) => values.flat())
+
+        // Include transactions per chain
+        txs[chainId] = txsByChain
+      }
+
+      setTxsAwaitingConfirmation(txs)
+      setLoading(false)
+    }
+    fetchAwaitingConfirmationTxs()
+  }, [ownedSafes, userAccount])
+
+  if (!userAccount) {
+    return <h3>Connect a wallet</h3>
+  }
+
+  if (loading || Object.keys(txsAwaitingConfirmation).length === 0) {
+    return <h3>Loading</h3>
+  }
+
+  return (
+    <List component="div">
+      {Object.entries(txsAwaitingConfirmation).map((chain) => {
+        const [chainId, transactions] = chain
+        const { shortName } = getChainById(chainId)
+        if (!transactions.length) return null
+
+        const aggregatedTxsBySafe: Record<string, number> = transactions.reduce((acc, val) => {
+          // Retrieve safeHash from the transactionId
+          const [, safeHash] = val.id.split('_')
+          if (!acc[safeHash]) {
+            return { ...acc, [safeHash]: 1 }
+          }
+          return { ...acc, [safeHash]: (acc[safeHash] += 1) }
+        }, {})
+
+        return Object.entries(aggregatedTxsBySafe).map((safeTxCount) => {
+          const [safeHash, count] = safeTxCount
+          return (
+            <TransactionToConfirm key={`${safeHash}`} href={`/${shortName}:${safeHash}/transactions/queue`}>
```

Comment:
```suggestion
            <TransactionToConfirm key={`${safeHash}`} href={`/app/${shortName}:${safeHash}/transactions/queue`}>
```

Reply:
We have `/app` set in `package.json`. You can use the `PUBLIC_URL` constant here.

```suggestion
            <TransactionToConfirm key={`${safeHash}`} href={`${PUBLIC_URL}/${shortName}:${safeHash}/transactions/queue`}>
```